File: daogen/projects/5253a8bd28afd2127079661b/Person.cs
using System;
using platform.dao.orm.attribute;

namespace platform.apptools.demo
{
    [Serializable]
    [Table(Name="Person")]
    public class Person
    {
        
        /// <summary>
        /// Name
        /// </summary>
        [Column(Name="Name")]
        public string Name { get; set; }
        
        /// <summary>
        /// Gender
        /// </summary>
        [Column(Name="Gender")]
        public int? Gender { get; set; }
        
        /// <summary>
        /// Age
        /// </summary>
        [Column(Name="Age")]
        public int? Age { get; set; }
        
        /// <summary>
        /// Telephone
        /// </summary>
        [Column(Name="Telephone")]
        public string Telephone { get; set; }
        
        /// <summary>
        /// Birth
        /// </summary>
        [Column(Name="Birth")]
        public DateTime? Birth { get; set; }
        
        /// <summary>
        /// Address
        /// </summary>
        [Column(Name="Address")]
        public string Address { get; set; }
        
        /// <summary>
        /// ID
        /// </summary>
        [Column(Name="ID"), PrimaryKey]
        public int? ID { get; set; }
        
    }
}

File: dal-tester/src/main/java/com/ctrip/platform/dal/tester/person/Person.java
package com.ctrip.platform.dal.tester.person;

import java.sql.Timestamp;

import com.ctrip.platform.dal.dao.DalPojo;

public class Person implements DalPojo {

	private Integer ID;

	public Integer getID() {
		return ID;
	}

	// (表示(不会被Verlocity解析
	public void setID(Integer ID) {
		this.ID = ID;
	}

	private String Address;

	public String getAddress() {
		return Address;
	}

	// (表示(不会被Verlocity解析
	public void setAddress(String Address) {
		this.Address = Address;
	}

	private String Telephone;

	public String getTelephone() {
		return Telephone;
	}

	// (表示(不会被Verlocity解析
	public void setTelephone(String Telephone) {
		this.Telephone = Telephone;
	}

	private String Name;

	public String getName() {
		return Name;
	}

	// (表示(不会被Verlocity解析
	public void setName(String Name) {
		this.Name = Name;
	}

	private Integer Age;

	public Integer getAge() {
		return Age;
	}

	// (表示(不会被Verlocity解析
	public void setAge(Integer Age) {
		this.Age = Age;
	}

	private Integer Gender;

	public Integer getGender() {
		return Gender;
	}

	// (表示(不会被Verlocity解析
	public void setGender(Integer Gender) {
		this.Gender = Gender;
	}

	private Timestamp Birth;

	public Timestamp getBirth() {
		return Birth;
	}

	// (表示(不会被Verlocity解析
	public void setBirth(Timestamp Birth) {
		this.Birth = Birth;
	}

}
